src/pydistcheck/distribution_summary.py

```diff
@@ -62,6 +62,10 @@ def num_directories(self) -> int:
     def num_files(self) -> int:
         return sum([1 for f in self.file_infos if f.is_file])
 
+    @property
+    def total_size_in_bytes_compressed(self) -> int:
+        return sum([f.compressed_size_bytes for f in self.file_infos])
```

Comment:
is this really the COMPRESSED size?

Because I see a size of like 104.79 MB being reported for `scikit_learn-1.1.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl`, which should only be like 30.5MB according to https://pypi.org/project/scikit-learn/#files.

<img width="535" alt="Screen Shot 2022-10-18 at 11 56 33 PM 1" src="https://user-images.githubusercontent.com/7608904/196601152-02bbf9a0-effd-4e12-96d0-57d84b1ddc65.png">

Reply:
yep, this was a bug! Fixed in #61